common/src/main/java/com/graphicsfuzz/common/util/UpgradeShadingLanguageVersion.java

```diff
@@ -46,14 +49,23 @@
 import java.io.PrintStream;
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import net.sourceforge.argparse4j.ArgumentParsers;
+import net.sourceforge.argparse4j.impl.Arguments;
 import net.sourceforge.argparse4j.inf.ArgumentParser;
 import net.sourceforge.argparse4j.inf.ArgumentParserException;
 import net.sourceforge.argparse4j.inf.Namespace;
 
 public class UpgradeShadingLanguageVersion extends ScopeTrackingVisitor {
 
+  // flag for disabling renames
```

Comment:
'flag' -> 'Flag'.

But it would be better to pass this as a command-line argument, by adding a --no-rename optional argument.

Reply:
Oh, I get it now - you're using this as a flag for bitmask stuff.  Can you update the comment to say something to that effect?

Reply:
Done. The reason I'm using a bit mask instead of touching the parameter parser everywhere is that when using the util from code, you don't (probably) want to drag the parameter parser along.